src/datasource/components/Logs/LogsCell.tsx

```diff
@@ -0,0 +1,325 @@
+import React from 'react'
+import { Log } from 'datasource/types'
+import { LogColumnType, LogColumn } from 'datasource/components/Logs/types'
+import  getLogTableContext from 'datasource/components/Logs/context'
+import { Button, useTheme2 } from '@grafana/ui'
+import { getTimeZone } from '@grafana/data'
+import { DateTime } from 'luxon'
+
+
+interface LogKeyValProps {
+    field: string,
+    val: any
+}
+
+
+const LogKeyVal = ({ field, val }: LogKeyValProps) => {
+    return (<div style={{display:'inline-block', paddingRight: '10px'}}>
+        <div style={{backgroundColor: useTheme2().isDark ? '#343741' : '#e6f1fa', display: 'inline', borderRadius: '6px', marginRight: '3px', padding: '2px 4px'}}>
+            {field + ":"}
+
+        </div>
+        <div style={{display: 'inline-block'}}>
+            {val}
+        </div>
+    </div>);
+}
+
+interface ExpandedLogKeyValProps {
+    field: string,
+    val: any
+}
+
+
+const ExpandedLogKeyVal = ({ field, val }: ExpandedLogKeyValProps) => {
+    return (
+        <tr>
+        <td style={{}}>
+            {field + ":"}
+
+        </td>
+        <td>
+            {val}
+        </td>
+    </tr>);
+}
+
+interface ExpandedDocumentProps {
+    log: Log,
+    index: number,
+    datasourceUid: string,
+    datasourceName: string
+}
+
+
+const ExpandedDocument  = ({ log, index, datasourceUid, datasourceName, }: ExpandedDocumentProps) => {
+    const { setSize, windowWidth } = getLogTableContext();
+    const root = React.useRef<HTMLDivElement>();
+    React.useEffect(() => {
+        if (root.current) {
+            setSize(index, root.current.getBoundingClientRect().height);
+        }
+    }, [windowWidth]);
+
+    const link = {
+        datasource: datasourceUid,
+        queries: [{
+            query: log.get("trace_id"), // TODO: This should be user configurable
```

Comment:
This is already configurable through the Grafana UI on the datasoruce, so we just need to lookup what's used there.

These links can be set on any field, as internal or external link, etc.

![image](https://github.com/slackhq/slack-kaldb-app/assets/771133/133a336b-2581-4816-9927-b1b0312ff9fe)